giskard_vision/core/detectors/perturbation.py

```diff
@@ -0,0 +1,114 @@
+import os
+from abc import abstractmethod
+from pathlib import Path
+from typing import Any, Sequence
+
+import cv2
+
+from giskard_vision.core.dataloaders.wrappers import FilteredDataLoader
+from giskard_vision.core.detectors.base import (
+    DetectorVisionBase,
+    IssueGroup,
+    ScanResult,
+)
+from giskard_vision.landmark_detection.tests.base import TestDiff
```

Comment:
I think we should move TestDiff in core instead of importing it from landmark_detection